draft-ietf-quic-http.md

```diff
@@ -549,20 +549,38 @@ SETTINGS frame MUST be processed in the order they appear, with no other frame
 processing between values. Unsupported parameters MUST be ignored. 
 
 Once all values have been processed, if the REQUEST_ACK flag was set, the 
-recipient MUST immediately emit a SETTINGS_ACK frame listing the identifiers 
-whose values were understood and applied. (If none of the values were 
-understood, the SETTINGS_ACK frame will be empty, but MUST still be sent.) Upon 
-receiving an SETTINGS_ACK frame, the sender of the altered parameters can rely 
-on the setting having been applied. 
-
-TODO:
-: The above text was written for HTTP/2 -- QUIC has cross-stream timing issues 
-here that need to be solved.
-
+recipient MUST immediately emit the following:
+
+ - On the connection control stream, a SETTINGS_ACK frame listing the 
+   identifiers whose values were understood and applied. (If none of the values 
+   were understood, the SETTINGS_ACK frame will be empty, but MUST still be
+   sent.) 
+
+ - On each request control stream which is not in the "half-closed (local)" or
+   "closed" state, an empty SETTINGS_ACK frame.
+
+The SETTINGS_ACK frame on the connection control stream contains the highest
+stream number which was open at the time the SETTINGS frame was received.  All
+streams with higher numbers can safely be assumed to have the new settings in
+effect when they open.
```

Comment:
I think that you have it.  Thanks for walking through it with me.  The sequential thing definitely needs work.  "Sequential" just means in order, which doesn't correspond to a mandate to use every number.